godtools/App/Features/Menu/Domain/UseCases/GetUserActivityStatsUseCase/GetUserActivityStatsUseCase.swift

```diff
@@ -23,46 +23,56 @@ class GetUserActivityStatsUseCase {
             textColor: Color.getColorWithRGB(red: 55, green: 167, blue: 160, opacity: 1),
             value: String(userActivity.lessonCompletions)
         )
         
         let screenSharesStat = UserActivityStatDomainModel(
             iconImageName: ImageCatalog.userActivityScreenShares.name,
-            text: localizationServices.stringForSystemElseEnglish(key: MenuStringKeys.Account.Activity.screenShares.rawValue),
+            text: getUserActivityText(stringKey: MenuStringKeys.Account.Activity.screenShares.rawValue, activityCount: userActivity.screenShares),
             textColor: Color.getColorWithRGB(red: 229, green: 91, blue: 54, opacity: 1),
             value: String(userActivity.screenShares)
         )
         
         let linksSharedStat = UserActivityStatDomainModel(
             iconImageName: ImageCatalog.userActivityLinksShared.name,
-            text: localizationServices.stringForSystemElseEnglish(key: MenuStringKeys.Account.Activity.linksShared.rawValue),
+            text: getUserActivityText(stringKey: MenuStringKeys.Account.Activity.linksShared.rawValue, activityCount: userActivity.linksShared),
             textColor: Color.getColorWithRGB(red: 47, green: 54, blue: 118, opacity: 1),
             value: String(userActivity.linksShared)
         )
         
         let languagesUsedStat = UserActivityStatDomainModel(
             iconImageName: ImageCatalog.userActivityLanguagesUsed.name,
-            text: localizationServices.stringForSystemElseEnglish(key: MenuStringKeys.Account.Activity.languagesUsed.rawValue),
+            text: getUserActivityText(stringKey: MenuStringKeys.Account.Activity.languagesUsed.rawValue, activityCount: userActivity.languagesUsed),
             textColor: Color.getColorWithRGB(red: 110, green: 220, blue: 80, opacity: 1),
             value: String(userActivity.languagesUsed)
         )
-
+        
         let sessionsStat = UserActivityStatDomainModel(
             iconImageName: ImageCatalog.userActivitySessions.name,
-            text: localizationServices.stringForSystemElseEnglish(key: MenuStringKeys.Account.Activity.sessions.rawValue),
+            text: getUserActivityText(stringKey: MenuStringKeys.Account.Activity.sessions.rawValue, activityCount: userActivity.sessions),
             textColor: Color.getColorWithRGB(red: 224, green: 206, blue: 38, opacity: 1),
             value: String(userActivity.sessions)
         )
         
         return [toolOpensStat, lessonCompletionsStat, screenSharesStat, linksSharedStat, languagesUsedStat, sessionsStat]
     }
+    
+    private func getUserActivityText(stringKey: String, activityCount: Int32) -> String {
+        
+        let formatString = localizationServices.stringForSystemElseEnglish(
+            key: stringKey,
+            fileType: .stringsdict
+        )
+        
+        return String.localizedStringWithFormat(formatString, activityCount)
```

Comment:
Hey @rachaelblue I'm good with leaving this PR as is for now since we're using LocalizationServices in the Account presentation.

Eventually we'll need to update this entire view interface strings to be based on the app language which will most likely effect the ```String.localizedStringWithFormat``` since that defaults to the current Locale (https://developer.apple.com/documentation/foundation/nsstring/1497301-localizedstringwithformat) and instead will need to use (https://developer.apple.com/documentation/foundation/nsstring/1497317-initwithformat).

Reply:
Hey @levieggertcru, yeah I was wondering when that should happen.  Didn't know if we had a ticket to rip out LocalizationServices altogether or not.

Reply:
Also I merged dev back into this branch, so conflicts are resolved.

Reply:
Okay nice I'm going to approve this PR.  I'd say go ahead and create a Jira task for updating the account activity to translate based on the app language.

Reply:
Sounds good, made a ticket here: https://jira.cru.org/browse/GT-2159